lib/utils.js

```diff
@@ -118,6 +118,20 @@ const utils = {
   },
 
   /**
+   * Grab the most recent PR
+   * @param {Config} config - the config object
+   * @param {Github} vcs - VCS interface
+   * @returns {Promise} a promise resolved with the most recent PR
+   */
+  getLastPr (config, vcs) {
+    return exec(`git log -50 --oneline | grep -m 1 "Merge pull request" | awk '{print $5;}'`).then((stdout) => {
```

Comment:
I don't think the `-50` is strictly necessary, since I'm fairly certain that the first commit will always be the one that was a `Merge pull request`